.github/workflows/setup-shared.yml

```
@@ -0,0 +1,67 @@
+        run: npm ci
+
+      - name: Build Assets
+        run: npm run build
+
+      - name: Cache Built Assets
+        uses: actions/cache@v4
+        with:
+          path: |
+            public/js
+            public/css
+          key: assets-${{ runner.os }}-${{ github.sha }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)